src/OrchardCore/OrchardCore.Abstractions/Modules/Extensions/StringExtensions.cs

```diff
@@ -17,5 +17,8 @@ public static byte[] ToByteArray(this string hex)
                 Select(x => Convert.ToByte(hex.Substring(x, 2), 16)).
                 ToArray();
         }
+
+        public static bool EqualsOrdinalIgnoreCase(this string first, string second)
```

Comment:
It returns false as string equals does

        public bool Equals([NotNullWhen(true)] string? value, StringComparison comparisonType)
        {
            if (ReferenceEquals(this, value))
            {
                CheckStringComparison(comparisonType);
                return true;
            }

            if (value is null)
            {
                CheckStringComparison(comparisonType);
                return false;
            }

Reply:
My bad, I will add a null check

Reply:
I think it's better to move it into a new helper instead of extension. This way we can place any string related helper in one place 

Str.Equals(string first, string second, StringComparison ops = StringComparison. OrdinalIgnoreCase)
{

}

Reply:
I put the extension in an existing `StringExtensions` in `OC.Absractions`.

Reply:
Guessing you did not like the idea of having helper method :)

Reply:
Yes I prefer extensions when possible ;)

Reply:
Str.Equals is much more appealing to me. Better readability, shorter and cleaner.  

Anyway, it does not matter much. Please approve or merge this PR if you are done.

Reply:
Okay I will approve when when I'm done but please retry it before merging